string mp3Id
string text
string lang
int32 period